scripts/alcor_services.sh

```diff
@@ -0,0 +1,158 @@
+
+    echo "Creating gateway_manager image with port 9015"
+    docker build -t gm $ALCOR_ROOT_DIR/services/gateway_manager/
+
+    echo "Creating ignite-11 image with ports 10800 10081 47100 47500"
+    docker build -t ignite-11 -f $ALCOR_ROOT_DIR/lib/ignite.Dockerfile $ALCOR_ROOT_DIR/lib
+}
+
+function start_alcor_containers()
+{
+    echo "Starting Alcor services "
+    docker run --name=ignite -p 10800:10800 -p 10801:10801 -p 47100:47100 -p 47500:47500 -v /tmp:/tmp -tid ignite-11 sh
+    docker run --net=host --name vpm -p 9001:9001 -v /tmp:/tmp -itd vpm
+    docker run --net=host --name snm -p 9002:9002 -v /tmp:/tmp -itd snm
+    docker run --net=host --name rm  -p 9003:9003 -v /tmp:/tmp -itd rm
+    docker run --net=host --name pim -p 9004:9004 -v /tmp:/tmp -itd pim
+    docker run --net=host --name mm  -p 9005:9005 -v /tmp:/tmp -itd mm
+    docker run --net=host --name pm  -p 9006:9006 -v /tmp:/tmp -itd pm
+    docker run --net=host --name nm  -p 9007:9007 -v /tmp:/tmp -itd nm
+    docker run --net=host --name sgm -p 9008:9008 -v /tmp:/tmp -itd sgm
+    docker run --net=host --name ag  -p 9009:9009 -v /tmp:/tmp -itd ag
+    docker run --net=host --name dpm -p 9010:9010 -v /tmp:/tmp -itd dpm
+    docker run --net=host --name eim -p 9011:9011 -v /tmp:/tmp -itd eim
+    docker run --net=host --name qm  -p 9012:9012 -v /tmp:/tmp -itd qm
+    docker run --net=host --name nam -p 9013:9013 -v /tmp:/tmp -itd nam
+    docker run --net=host --name gm  -p 9015:9015 -v /tmp:/tmp -itd gm
+}
+
+function stop_alcor_containers()
+{
+    echo "Stopping all alcor services"
+    docker container stop ignite
+    docker container stop vpm
+    docker container stop snm
+    docker container stop rm
+    docker container stop pim
+    docker container stop mm
+    docker container stop pm
+    docker container stop nm
+    docker container stop sgm
+    docker container stop ag
+    docker container stop dpm
+    docker container stop eim
+    docker container stop qm
+    docker container stop nam
+    docker container stop gm
+}
+
+function remove_alcor_containers()
+{
+    echo "Removing Alcor containers"
+    docker container rm ignite
+    docker container rm vpm
+    docker container rm rm
+    docker container rm pim
+    docker container rm qm
+    docker container rm nm
+    docker container rm mm
+    docker container rm sgm
+    docker container rm ag
+    docker container rm dpm
+    docker container rm eim
+    docker container rm pm
+    docker container rm snm
+    docker container rm nam
+    docker container rm gm
+}
+
+function command_help()
+{
+    echo " Choose following options:"
+    echo
+    echo "  -b For building Alcor microservice's images"
+    echo "  -a For running Alcor microservices containers"
+    echo "  -o For stopping Alcor microservices containers"
+    echo "  -r For removing Alcor microservice containers"
+    echo "   please pick one from [b], [a], [o] or [r]"
+    echo
```

Comment:
@kiran1048 Can you create another option for entire processes including stop dockers (if exist), rm images (if exist), building, and running?